devlib/bin/scripts/shutils.in

```diff
@@ -177,6 +177,15 @@ cgroups_tasks_in() {
 	exit 0
 }
 
+cgroup_trace_attach_task() {
+        DST_ROOT=${1}
+        DST_PATH=${2}
+        TASKS_FILE=${3}
+
+        cat $TASKS_FILE | while read PID; do
+            echo "cgroup_attach_task_devlib: dst_root=$DST_ROOT dst_path=$DST_PATH pid=$PID" > /sys/kernel/debug/tracing/trace_marker
```

Comment:
Yes I believe, only the extra fields are dropped but the existing fields have the same format, so that's not an issue